control/control.yaml

```diff
@@ -176,5 +176,14 @@ components:
           type: string
           enum:
           - lacp_member_state
```

Comment:
If changing the enum sequence does not break compatibility, I would prefer to keep mostly used events first - with the following sequence: 
      protocol_state
      route_state
      lacp_member_state
      send_ping